pallets/nfts/src/lib.rs

```diff
@@ -693,6 +736,24 @@ pub mod pallet {
 		CollectionNotEmpty,
 		/// The witness data should be provided.
 		WitnessRequired,
+		/// The account owns zero items in the collection.
+		NoItemOwned,
```

Comment:
Note: Tested in
- Force-approving an unknown collection: https://github.com/r0gue-io/pop-node/pull/387/files#r1881265589